packages/backend/src/accounting/tigerbeetle/transfers.ts

```diff
@@ -154,3 +166,26 @@ export async function createTransfers(
     }
   }
 }
+
+export async function getAccountTransfers(
+  deps: ServiceDependencies,
+  id: string | number,
+  limit: number = 100_000
+): Promise<GetLedgerTransfersResult> {
+  const account_id = toTigerBeetleId(id)
+  const filter: AccountFilter = {
+    account_id,
+    timestamp_min: 0n,
+    timestamp_max: 0n,
+    limit, //100_000
+    flags: AccountFilterFlags.credits | AccountFilterFlags.debits
```

Comment:
We do not currently split between debits / credits. Fetch all for account, we may want to change that in future.